rootfs/api/settings/production.py

```diff
@@ -24,8 +24,6 @@
     'security.W008'
 ]
 
-CONN_MAX_AGE = 60 * 3
```

Comment:
Does this need to be added back. I think this is a different `CONN_MAX_AGE` env variable?